### ISSUE TITLE
Conscript Error

### ISSUE THREAD

**Issue by dghant1024** [foundweekends/conscript#72]
System: Windows 8 64-bit
java version "1.7.0_76"
javac version "1.7.0_71"

------------------------------------------------------------------------

By simply attempting to launch the 'conscript-0.4.4.jar' I receive the  error message below: 

![image](https://cloud.githubusercontent.com/assets/2386461/7320040/f6360540-ea67-11e4-93a2-27fefe6d6f49.png)

Other attemps from command line I do: 

C:\Users\[username]>java -jar -Dhttps.proxyHostwww-proxy.sct.com  -Dhttps.proxyPort8080  conscrip
t-0.4.4.jar

AND

C:\Users\[username]>java -jar conscript-0.4.4.jar

Which results in the error below: 

Error: Unable to access jarfile conscript-0.4.4.jar

Also, can someone please confirm that I'm using the command below properly:

C:\Users\[username]>java -jar -Duser.home=C:/Users/dghant/ -Dhttps.proxyHostwww-proxy.server.com  -Dhttps.proxyPort8080 conscript-0.4.4-1.jar

Apparently, I'm missing something simple.  Please advise!

**Comment by klimchuk** [comment]
I have the same issue 'Error: Unable to access jarfile conscript-0.4.4.jar' with Windows 8.1 64 bit and Java 1.8.0_45
It seems to be impossible to run jar

**Comment by saritasa** [comment]
the same issue with totally fresh install on windows 10

**Comment by nartamonov** [comment]
conscript is not usable at all, the same issue on Ubuntu:

```
Error downloading sbt-launch-0.13.0: java.util.concurrent.ExecutionException: dispatch.StatusCode: Unexpected response status: 404
```

**Comment by saritasa** [comment]
It's usable with dance around :). The problem is that sbt-launch-0.13.0 is totally missed from sbt site. So setup procedure fails trying to downloading it (it then produces zero byte file)

You need to install the latest version (sbt), then just copy and rename the file to sbt-launch-0.13.0 and things will work properly.

I did it on windows and it worked like a charm. No issues

**Comment by AlexHolly** [comment]
I get this on Ubuntu with your workaround.
```
Error: Unable to access jarfile /root/.conscript/sbt-launch.jar
cs: invalid option: -
Error: Unable to access jarfile /root/.conscript/sbt-launch.jar
```

**Comment by BillOTei** [comment]
for some reason, it appears you need the 2 files: sbt-launch-0.13.0.jar and a copy called sbt-launch.jar in the .conscript directory. Both comming from the sbt install dir.

**Comment by AlexHolly** [comment]
Thanks. now it works somehow.
The UI still shows the error and the terminal shows 
```
sbt launcher version 0.13.7
sbt launcher version 0.13.7
sbt launcher version 0.13.7
```
Why is no one fixing this issue?

**Comment by RutujaShirali** [comment]
Still does not work for me! Any permanent fix for this?

**Comment by paulkav1** [comment]
On Windows 10, Java 8:
install SBT. Copy sbt-launch.jar to c:/Users/username/.conscript
also copy it to same place as sbt-launch-0.13.0.jar
Now add cs to PATH

**Comment by frozenspider** [comment]
@paulkav1 Thanks, that worked!
Quite unfortunate that noone's fixing the issue though

**Comment by eed3si9n** [comment]
I'm guessing this is now fixed by #75
